### PR TITLE
header-component

### PR DESCRIPTION
**Pull request by MikhailBiltsevich**
`Header` displays on the screenshot below (user have authentificated)

Снимок экрана (30)

But I have a question to @DieselWolf Should I use a `<ButtonComponent>` instead `<button>` in this header? (On the second thought I think **yes**)

### CODE REVIEW

File: src/components/HeaderComponent/HeaderComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  function signOutHandler(): void {
+  }
+
+  return (
+    <header className='sticky row'>
+      <span className='logo col-sm-12 col-md-2'>quizes</span>
+      {props.user &&
+        <>
+          <span className='button col-sm-12 col-md'>{props.user.login}</span>
+          <ButtonComponent classNames={['button', 'col-sm-12', 'col-md-2']} handlerClick={signOutHandler} text='Sign out' />
```
Comment (DieselWolf) [inline src/components/HeaderComponent/HeaderComponent.tsx:20]: If you decide to use Button component on your Header Component, you must create special section for Button, it shouldn't be hard coded component.
The same for text and other components which you will use in header

**Review by DieselWolf — CHANGES_REQUESTED** [review]

File: src/components/HeaderComponent/HeaderComponent.tsx (hunk 1 above)
Comment (MikhailBiltsevich) [reply src/components/HeaderComponent/HeaderComponent.tsx:20]: It's about only components in Header? Do you mean that I have to wrap each component that located in Header?

File: src/components/HeaderComponent/HeaderComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+interface IProps {
+  content?: JSX.Element
```
Comment (DieselWolf) [inline src/components/HeaderComponent/HeaderComponent.tsx:2]: I'm not sure about this type
I think it should related to component

**Review by DieselWolf — CHANGES_REQUESTED** [review]

**Comment by MikhailBiltsevich** [comment]
Hi! I did some mistake, when commit changes by `git commit --ammend` after pushing branch into remote repository. I used `any` type for `content` prop. So we can use string or html-template for example.

**Review by DieselWolf — APPROVED** [review]

**Pull request merged**